src/lib/OpenEXR/ImfCompression.cpp

```diff
@@ -192,6 +198,7 @@ static const std::map<std::string, Compression> CompressionNameToId = {
     {"b44a", Compression::B44A_COMPRESSION},
     {"dwaa", Compression::DWAA_COMPRESSION},
     {"dwab", Compression::DWAB_COMPRESSION},
+    {"ht256", Compression::HT256_COMPRESSION},
```

Comment:
```suggestion
    {"ht", Compression::HT_COMPRESSION},
```
Rename `ht256` to `ht` 256-scanline chunks.

Reply:
Does this still need to be resolved? "ht256" still appears in other places, such as in `CompressionDesc` on line 180 above, does that need to be consistent?